src/typescript/sdk/src/indexer-v2/json-bigint.ts

```diff
@@ -25,7 +25,7 @@ const tryWithFallbackParse = (parser: (v: any) => any) => (v: any) => {
     return v;
   }
 };
-const parseFloat = (v: any) => Big(v).toString();
+const parseFloat = (v: any) => (v === null ? null : Big(v).toString());
```

Comment:
```suggestion
const parseFloat = (v: any) => Big(v).toString();
```

Since the candlestick prices are no longer `Nullable`.